src/app/payment/PaymentForm.tsx

```diff
@@ -215,14 +215,8 @@ const PaymentMethodListFieldset: FunctionComponent<PaymentMethodListFieldsetProp
         setSubmitted,
     ]);
 
-    const legend = useMemo(() => (
-        <Legend>
-            <TranslatedString id="payment.payment_method_label" />
```

Comment:
I think we could remove this from [the translations file](https://github.com/bigcommerce/checkout-js/blob/2fd2e57016e00ec77eded07759af7720749eafd6/src/app/locale/translations/en.json#L223) too? cc @bigcommerce/checkout

Reply:
good catch @TomA-R